src/index.js

```diff
@@ -68,6 +68,7 @@ export const withBreadcrumbs = routes => Component => withRouter(props =>
     ...props,
     breadcrumbs: getBreadcrumbs({
       pathname: props.location.pathname,
-      routes,
+      routes, 
+      location: props.location,      
```

Comment:
Let's just pass `location` to `getBreadcrumbs` and let that method pull `pathname` from it instead of passing both fields.

Reply:
I agree. But this would be a breaking change. I'm fine if you're OK with it.

Reply:
Good call, yea I'm alright with it as I don't think many people are using the exported `getBreadcrumbs` function. I can tag it as a minor patch and call it out in the readme/release notes.

Reply:
Oh well, just saw this. Feel free to edit the readme/package.json! :)